packages/@aws-cdk/aws-kms/test/test.key.ts

```diff
@@ -65,6 +65,66 @@ export = {
     test.done();
   },
 
+  'default with no retention'(test: Test) {
+    const app = new App();
+    const stack = new Stack(app, 'TestStack');
+
+    new EncryptionKey(stack, 'MyKey', { retainKey: false });
+
+    expect(app.synthesizeStack(stack.name)).to(exactlyMatchTemplate({
```

Comment:
Can you tighten this assertion a little? Use `haveResource` and only include the `DeletionPolicy`? Makes the test less sensitive to changes

Reply:
So loosen it :)